test/integration/testdata/egress-rule-google.yaml.tmpl

```diff
@@ -0,0 +1,11 @@
+apiVersion: config.istio.io/v1alpha2
+kind: EgressRule
+metadata:
+  name: google
+spec:
+  destination:
+      service: "*google.com"
+  ports:
+      - port: 443
+        protocol: https
+  use_egress_proxy: false
```

Comment:
May be enable the egress proxy for one of these?

Reply:
using egress proxy is not implemented - we should implement it in 0.3. It depends on either using nginx or on adding forward proxy functionality to envoy